### PR TITLE
Updated review criteria

### PR DESCRIPTION
**Pull request by dalippa**
@deniederhut @cbcunc @stargaser 

Apologies for the lateness, but I had some personal and work surprises come up, which delayed my additions to the reviewer guidelines. Please take a look and provide feedback.

### CODE REVIEW

**Review by deniederhut — APPROVED** [review]

**Comment by deniederhut** [comment]
This looks great! Thanks @dalippa

File: review_criteria.md [hunk 1]
```diff
@@ -71,3 +71,34 @@ review criteria are enthusiastically welcomed via discussion or pull request.
...
+  providing suggestions for the first few incidents and then referencing them later
+  will help authors while reducing reviewer time.
+  * Instead of: "Sentence is too long and wordy."
+  * Consider: "The entire paragraph is comprised of 2 sentences, making it hard to follow.
+    Either consolidate it into the previous or next paragraph or flesh out the existing
+    paragraph more."
+  * Ideally (and time permitting): Provide phrasing and word fragments such as "Consider
+    rewording first sentence to avoid ending with a preposition. Instead of `First, we
+    identified essays that we wanted to run analysis on`, consider `First, we identified
+    essays for analysis`".
```
Comment (stargaser) [inline review_criteria.md:90]: Could we make use of Github's new feature to make a suggestion, in inline comments?

**Review by stargaser — COMMENTED** [review]
The new feedback section looks great! I left an inline comment to ask if we can make use of the recently-added Suggestions feature in Github. I've used it for code but not for `.rst` files.
